runtime/compiler/control/J9Options.cpp

```diff
@@ -1989,15 +1989,23 @@ J9::Options::fePreProcess(void * base)
          std::random_device rd;
          std::mt19937_64 rng(rd());
          std::uniform_int_distribution<uint64_t> dist;
+         // We want to generate a non-zero client UID for client mode
```

Comment:
If jitserver jvm is running in server mode, this section ensures client UID is non-zero and stores it to persistent info. 
Under below `else` block, client UID is set to zero if running as server mode.